providers/record.rb

```diff
@@ -79,7 +79,7 @@ def record_value_or_alias_attributes
 end
 
 action :create do
-  require 'fog'
+  require 'fog/aws/dns'
```

Comment:
You don't need to specify `dns`.

If you require `fog/aws` it will define all services using autoload and load only the ones you use that in this case will be `DNS`.